appservice/src/createAppService/createAppService.ts

```diff
@@ -52,21 +64,31 @@ export async function createAppService(
                     learnMoreLink: 'https://aka.ms/Cfqnrc'
                 }
             ));
+            promptSteps.push(new AppServicePlanListStep());
+            promptSteps.push(new LocationListStep());
             break;
         case AppKind.app:
+            if (advancedCreation) {
+                promptSteps.push(new ResourceGroupListStep());
+                promptSteps.push(new SiteOSStep());
+                promptSteps.push(new SiteRuntimeStep());
+                promptSteps.push(new AppServicePlanListStep());
+                promptSteps.push(new LocationListStep());
+            } else {
+                if (workspace.workspaceFolders.length === 1) {
```

Comment:
strictNullChecks is turned off in the appservice package. Until we fix it (#111) you need to be extra careful. For example, I think workspaceFolders can be undefined